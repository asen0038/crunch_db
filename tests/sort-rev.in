set a 7 2 8 9
sort a
rev a
List entries
bye
