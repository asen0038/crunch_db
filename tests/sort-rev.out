>
ok

>
ok

>
ok

>
a [9 8 7 2]

>
bye
